notebooks/02_analyses/hatesonar_analysis.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>@cdolfi can you do a print out, such as a <code>df.head()</code> after this step to view the contents of the dataframe after the previous processing step and add some conclusion on how do you judge the performance of this hate_speech method?</p><p> </p><p>It would be interesting to see how many rows are actually being classified as 'Hate Speech' and 'Offensive Language' by Hatesonar</p><p>A <code>df.Top.unique()</code>  could show us what all categories it detects.</p><p> </p><p>It would be nice to see how many and which emails are being classified as hate speech or offensive language. Maybe you can try the following </p><p><code></code><code></p><p>offensive_df = df.loc[df['Top'] == 'offensive_language']</p><p>offensive_df.head()</p><p>len(offensive_df)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>